webserver/__init__.py

```diff
@@ -10,6 +10,10 @@ def create_postgres():
     from postgres_connection import init_postgres_connection
     return init_postgres_connection(current_app.config['SQLALCHEMY_DATABASE_URI'])
 
+def create_redis():
```

Comment:
In fact, it's already being done: https://github.com/metabrainz/listenbrainz-server/blob/master/webserver/__init__.py#L32. Is there some kind of issue with that?